test/functional/api/cas/ioclass_config.py

```diff
@@ -148,7 +151,7 @@ def create_ioclass_config(
         )
     if add_default_rule:
         output = TestRun.executor.run(
-            f'echo "0,unclassified,22,1" >> {ioclass_config_path}'
+            f'echo "0,unclassified,22,1.00" >> {ioclass_config_path}'
```

Comment:
You can use DEFAULT variables in here so it could be easier to update when some of them will be changed in the future